### PR TITLE
Update: Support .dwg files in the DocumentViewer

### PR DESCRIPTION
**Pull request by ikerr**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @ikerr, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: src/lib/extensions.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const CODE_EXTENSIONS = [
     'css',
     'cxx',
     'diff',
+    'dwg',
```
Comment (jeremypress) [inline src/lib/extensions.js]: CODE_EXTENSIONS are also used by the TextLoader, I think it's possible that the wrong viewer might try to load a dwg depending on order of an array.

Comment (ikerr) [reply src/lib/extensions.js]: Ah, so should this go in DOCUMENT_EXTENSIONS instead?

Comment (jeremypress) [reply src/lib/extensions.js]: sounds good. Can DWGs have multiple pages or are they single page file types?

Comment (ikerr) [reply src/lib/extensions.js]: They can have multiple pages.

**Pull request merged**